invalid_sample_generator.py

```diff
@@ -0,0 +1,153 @@
+import requests
+import csv
+import xml.etree.ElementTree as ET
+import json
+import random
+import os
+
+
+biosample_field_vals = {
+    "collection date":"collection_date"
+}
+
+mandatory_error_file_extension = '_mandatory.json'
+enum_error_file_extension = '_enum.json'
+def parse_csv_to_dict(file_path):
+    result_dict = {}
+    with open(file_path, mode='r') as file:
+        csv_reader = csv.DictReader(file)
+        for row in csv_reader:
+            key = row['accession']
+            value = row['checklist']
+            result_dict[key] = value
+    return result_dict
+
+def download_ena_checklists():
```

Comment:
You could have used the json schema already in the repo in `schema` directory. It would have saved some trouble.

Reply:
Schemas available in schema directory are ena sample schema not checklist schema.

Reply:
I think the schema directory contains the checklist json schema, either in ENA format or BioSamples format. Do you see something else?